Automattic/pocket-casts-android
# Reduce APK size by resizing the podcast images used

## Description
This uses webp images for podcasts images to reduce apk size.

Fixes #791

## Testing Instructions
1. Fresh install the app
2. Notice that images are loaded properly on the onboarding launch screen

## Screenshots or Screencast 

<img src="https://user-images.githubusercontent.com/1405144/221481241-1b7fc5d2-96f5-4595-9a96-29f967802117.png" width = 320/>
